components/chat/ConversationList.vue

```diff
@@ -1,27 +1,83 @@
 <template>
   <div>
-    <span>Conversations ids</span>
-    <div v-for="(id, index) in conversationsIds" :key="index">
-      <NuxtLink :to="`/chat/conversations/${id}`" class="text-blue-livementor">
-        {{ id }}
+    <div class="text-gray-500 font-bold my-4 text-2xl text-center">
+      Conversations
+    </div>
+    <div class="items">
+      <NuxtLink
+        v-for="(conversation, index) in conversations"
+        :key="`conversation-${conversation.id}`"
+        :to="`/chat/conversations/${conversation.id}`"
+      >
+        <div
+          :class="[
+            'items__tile flex items-center p-6 bg-white rounded-l-full',
+            {
+              'hover:bg-gray-100': conversation.id !== currentRouteId,
+              'rounded-br-full':
+                conversations[index + 1] &&
+                conversations[index + 1].id === currentRouteId,
+              'items__tile--active bg-indigo-100':
+                conversation.id === currentRouteId,
+              'rounded-tr-full':
+                conversations[index - 1] &&
+                conversations[index - 1].id === currentRouteId,
```

Comment:
I needed to test with the current router id param because I couldn't find anything close to [vue-router v-slot-api](https://router.vuejs.org/api/#v-slot-api-3-1-0) with NuxtLink.

Reply:
https://github.com/nuxt/nuxt.js/blob/2cf68b6019f594dfadec77ac7de50d7d534143a9/lib/app/components/nuxt-link.js
Si tu regardes comment est render le component nuxt-link tu vas te rendre compte que c'est exactement la meme chose que le router-link

Reply:
Yes, merci ça fonctionne nickel, je ne m'étais pas douté de ça 😄